src/application.ts

```diff
@@ -11,6 +11,7 @@ import multer from 'multer';
 import path from 'path';
 import {FILE_UPLOAD_SERVICE, STORAGE_DIRECTORY} from './keys';
 import {MySequence} from './sequence';
+const fs = require('fs')
```

Comment:
why const ? if you're on ts isn't it supoose to be:
import fs from 'fs' ?

Reply:
@ShirMon

Reply:
Done, it doesn't make much a difference but the ts standard is import from so there you go.